pkg/controller/wip/controller_test.go

```diff
@@ -1031,18 +1031,43 @@ func TestReconcileBindingWithParameters(t *testing.T) {
 	if len(updateObject.Spec.Parameters.Raw) == 0 {
 		t.Fatalf("Parameters was unexpectedly empty")
 	}
-	// TODO(vaikas): Implement the storing logic in the fake Binding Client so that it stores
-	// something meaningful there. For now, it just stores a struct making the validation a bit
-	// wonky.
-	if _, ok := fakeBindingClient.Bindings[instanceGUID+":"+bindingGUID]; !ok {
+	if b, ok := fakeBindingClient.Bindings[instanceGUID+":"+bindingGUID]; !ok {
```

Comment:
should this be calling `bindingsMapKey()` instead of hard coding the strcat so that if bindingsMapKey() changes how it calcs the key then the test won't break?